pkg/querier/queryrange/codec.go

```diff
@@ -413,6 +413,9 @@ func (Codec) DecodeRequest(_ context.Context, r *http.Request, _ []string) (quer
 			Step:         0,
 			TargetLabels: req.TargetLabels,
 			AggregateBy:  req.AggregateBy,
+			CachingOptions: queryrangebase.CachingOptions{
```

Comment:
should we document that these are now not cached by default? (and caching of them is not configurable by the user if we're hard coding this here?)

Reply:
oh, hmm, I missed this.... how are they not cached by default? My intent here was to keep the behavior but allow them to not be cached by setting the `Cache-Control` header.

Reply:
Nevermind, I misunderstood based on the field name. I assumed `Disabled` would only be present if the results caching _was_ disabled. Missing the historical context of why the field is named `Disabled` and not something like `ShouldCache`.